IdentityCore/src/controllers/broker/MSIDBrokerInteractiveController.m

```diff
@@ -129,6 +129,7 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
 + (BOOL)canPerformRequest:(MSIDInteractiveRequestParameters *)requestParameters
 {
 #if AD_BROKER
+    (void)requestParameters;
```

Comment:
shall we use `#pragma unused(requestParameters)` as the same style is used elsewhere?

Reply:
done